Zend/tests/pipe_operator/function_not_found.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Pipe operator throws normally on missing function
+--FILE--
+<?php
+
+try {
+    $res1 = 5 |> '_test';
+}
+catch (Throwable $e) {
+    printf("Expected %s thrown, got %s", Error::class, get_class($e));
+}
```

Comment:
```suggestion
catch (Throwable $e) {
    echo $e::class, ": ", $e->getMessage(), PHP_EOL;
}
```
To assert on the class + message. Or just don't catch the exception.

Same elsewhere.